admin/brands/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+
+from osf.models import Brand
+from django.forms.widgets import TextInput
+
+class BrandForm(forms.ModelForm):
```

Comment:
It would be really nice if we could add some help text to the form that told the appropriate dimensions of the images you are linking to.

Reply:
^ +1 to that idea! maybe ping Ash for the dimensions?